tests/store/tracking/test_sqlalchemy_store.py

```diff
@@ -939,6 +939,12 @@ def test_search_params(self):
         filter_string = "params.p_b = 'ABC'"
         six.assertCountEqual(self, [r2], self._search(experiment_id, filter_string))
 
+        filter_string = "params.generic_2 LIKE '%other%'"
+        six.assertCountEqual(self, [r2], self._search(experiment_id, filter_string))
```

Comment:
Can we have a negative case for `LIKE '%Other%'`?  This should return no results (or the same results if sqlite, with a comment).

Reply:
Actually the sqlalchemy store tests run on sqllite, so this would return the same results